### DISCUSSION TITLE
Can anyone help me find darbouka.ts?

### DISCUSSION THREAD

**Discussion by klangmensch** [acids-ircam/RAVE#114] [Q&A]
Sorry, I'm a total noob. I got Rave nn~ running inside max msp with the wheel.ts model. Can someone tell me where to find darbouka.ts from the tutorial video and maybe some additional models? Much appreciated!

**Comment by smithee771** [comment]
Hi, I assume that darbouka.ts is a model trained by the author, not available for public. You can change with the name of your custom _rt.ts file.
Hope that helps
